catroid/src/main/java/org/catrobat/catroid/ui/BaseExceptionHandler.java

```diff
@@ -45,6 +47,7 @@ public BaseExceptionHandler(Context context) {
 	public void uncaughtException(Thread thread, Throwable exception) {
 		CrashReporter.storeUnhandledException(exception);
 		preferences.edit().putBoolean(RECOVERED_FROM_CRASH, true).commit();
+		Log.e(TAG, "uncaughtException: ", exception);
```

Comment:
Exactly! Thanks!

Oh, please move this line up to the top of the method. Logging the error should be the first statement imo.